impress-org/givewp
# Chore: Update list of supported add-ons

## Description



This PR updates the list of supported add-ons according to https://givewp.com/documentation/core/givewp-3-0-compatibility-guide/ (as of the time of this PR).

## Testing Instructions



Check that supported add-ons, when installed and active, are listed in the Upgrade notice.
